app/models/user.rb

```diff
@@ -33,9 +33,14 @@ class User < ApplicationRecord
   # Additional validation for emails, :validatable has already added basic validation
   validate :email_uniqueness, on: :create
 
+  # Required fields
   validates :name, presence: true
+  validates :email, presence: true
```

Comment:
This is required by devise, but the user can delete it again. I don't think we should allow that.